forms-flow-theme/src/style-classes/List.scss

```diff
@@ -0,0 +1,126 @@
+.grid-forms {
+    .col-md-4 {
+        flex: 0 0 50% !important;
+        max-width: 50% !important;
+        padding-top: 15px;
+    }
+
+    .col-md-8 {
+        flex: 0 0 50% !important;
+        max-width: 50% !important;
+    }
+}
+
+.custom-grid {
+    .col {
+        min-height: 50px;
+        text-align: start;
+        padding-top: 15px;
+        height: auto;
+        word-break: break-all;
+    }
+
+    .list-group-item {
+        border-top: 1px solid #ced4da !important;
```

Comment:
We would need to replace the hardcoded colour to use from variables , similarly any other place we can make dynamic